Automattic/jetpack
# Subscribe Block: Add pre/post-publish panels

Built off of #22694, will change base branch to `master` once it has been merged. Feature requested by @mtias.

#### Changes proposed in this Pull Request:

* Initializes a `subscriptions` Gutenberg plugin to enable pre/post publish panels.
* Adds pre/post publish sidebar panels indicating the number of subscribers.
<table>
<tr>
<td><img width="262" alt="Screen Shot 2022-02-15 at 4 07 22 PM" src="https://user-images.githubusercontent.com/4044428/154166093-20fe1cfc-82e1-4010-8f63-a241ec58ebae.png">
<td><img width="262" alt="Screen Shot 2022-02-15 at 4 07 09 PM" src="https://user-images.githubusercontent.com/4044428/154166100-c131fdb6-64cb-4bc0-b9ee-dcb6144cefe2.png">
</table>

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Apply this change to your connected test site.
* Enable subscriptions in `Jetpack` -> `Settings` -> `Discussions`.
* Write a new post and press "Publish."
* Ensure that the new pre-publish panel is present.
* Press "Publish" again to confirm.
* Ensure that the new post-publish panel is present.
